README.md

```diff
@@ -126,6 +127,54 @@ client = Redis.new(semian: {
 })
 ```
 
+#### Net::HTTP
+For the `Net::HTTP` specific Semian adapter, since many external libraries may create
+HTTP connections on the user's behalf, the parameters are instead provided
+by calling specific functions in `Semian::NetHTTP`, perhaps in an initialization file.
+
+To give Semian parameters, assign a `proc` to `Semian::NetHTTP.semian_configuration`
+that takes two parameters, `host` and `port like `127.0.0.1` and `80` or `github_com` and `80`,
+and returns two values, a `semian_identifier` and a `Hash` with the keys as follows.
+
+```ruby
+SEMIAN_OPTIONS = { tickets: 1,
+                   success_threshold: 1,
+                   error_threshold: 3,
+                   error_timeout: 10 }
+Semian::NetHTTP.semian_configuration = proc do |host, port|
+  semian_identifier = "nethttp_#{host}_#{port}"
+  # Let's make it only active for github.com
+  if semian_identifier == "nethttp_github.com_80"
+    [semian_identifier, SEMIAN_OPTIONS]
```

Comment:
Could we instead return a hash like the following, and have the identifier be optional (use default).
```
{identifier: semian_identifier, options: SEMIAN_OPTIONS}
```
It makes it a bit clearer instead of using an array.